test/src/k-provider/ovp/provider.spec.js

```diff
@@ -474,6 +474,81 @@ describe('getEntryListConfig', function() {
   });
 });
 
+describe('getPlaybackContext', () => {
+  let provider, sandbox;
+  const partnerId = 1068292;
+  const ks =
+    'NTAwZjViZWZjY2NjNTRkNGEyMjU1MTg4OGE1NmUwNDljZWJkMzk1MXwxMDY4MjkyOzEwNjgyOTI7MTQ5MDE3NjE0NjswOzE0OTAwODk3NDYuMDIyNjswO3ZpZXc6Kix3aWRnZXQ6MTs7';
+  const playerVersion = '1.2.3';
+
+  afterEach(() => {
+    sandbox.restore();
+    MultiRequestBuilder.prototype.execute.restore();
+  });
+
+  it('should entryId be equal to the requested entryId', done => {
```

Comment:
don't think this is proper english

Reply:
Also, what is the test checking exactly? this is a black box API response responsibility, and you don't really know what the API will return.
Unless missing something, you can remove this test